### DISCUSSION TITLE
How get dubbo service registry namespace on dubbo filter ?

### DISCUSSION THREAD

**Discussion by Cloud-Yao** [apache/dubbo#13982] [Question]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SPI Extensions (apache/dubbo-spi-extensions)

### Details

Q1：I am creating dubbo filter , i want get current dubbo service registry namespace (registry on nacos) , as so far, I don't find namespace data ?  In other words, how to find invoker registry namespace on dubbo filter method ?

Q2:  I am using newest dubbo rest (version 3.3-beta), as consumer,  want to call provider method ,  how to deliver request header parameter ?  I using "Attachment"  deliver request header , it' s right ?  And I also find , I use "Attachment" deliver request header , then header key include prefix "rest-service", how remove "rest-service" ?

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
Q1: It would better for you to add a `Router` to add cutomized `Attribute` or `Attachment` in `Invocation`. Then, in `Filter` you can get it.

Q2: You can tri use triple protocol. In 3.3.0-beta.2, we have added the native http support in it.
